guides/pages/how-to-synchronize-your-yjs-storage-data-to-a-planetscale-mysql-database.mdx

```diff
@@ -0,0 +1,584 @@
+---
+meta:
+  title:
+    "How to synchronize your Yjs Storage data to a PlanetScale MySQL database"
```

Comment:
"How to synchronize your stored Yjs document data to a..." is more aligned with how we name things else where.

Liveblocks Document = data stores for collaborative documents.
We have two data stores: Liveblocks Storage and Liveblocks Yjs.

Reply:
Whoops, meant to change this. Thanks!